README.md

```diff
@@ -17,13 +17,29 @@
     from channels_yroom.consumer import YroomConsumer
 
     class TextCollaborationConsumer(YroomConsumer):
+        room_name_prefix: str = "yroom"
+        room_url_kwargs: str = "room_name"
+
+        # or
+
         def get_room_group_name(self) -> str:
             """
             Determine a unique name for this room, e.g. based on URL
             """
             room_name = self.scope["url_route"]["kwargs"]["room_name"]
             return "textcollab_%s" % room_name
 
+
+        # use hooks or override the default
+
+        # hook:
+
+        async def pre_connect(self) -> None:
+            user = self.scope["user"]
+            if not user.is_staff:
+                await self.close()
```

Comment:
Here's a problem: how to return in the parent function to not execute the next code instructions?